### ISSUE TITLE
Some questions about preprocessing operations

### ISSUE THREAD

**Issue by 531126085** [JihyongOh/FISR#6]
First of all, thank you very much for your public and maintenance of the project.
I see that your input data is PWC-Net processed for continuous frames, then warp, and then send these data to the network together. Whether these two steps of preprocessing can be regarded as a frame interpolation operation. The subsequent network just played a super-resolution effect?
So I still have to complete the tasks of super resolution and frame interpolation in two steps. I do n’t know how you think about this problem

**Comment by JihyongOh** [comment]
Hi, @531126085,
As described in the section "Network Architecture" of the paper, 

"to effectively handle large motion and occlusions, the bidirectional optical flow maps **(calculated by the pre-trained PWC-Net)** and the corresponding warped frames are stacked with the input frames,"

they are meaningful hints to the FISRnet, it is not a **preprocessing** concept. The FISRnet **itself** works for both frame interpolation and super resolution simultaneously. 
Without those steps, FISRnet still works well as shown in Table 3 ("Baseline", "+Temporal Loss", "+Multi-scale"). FISR
